### DISCUSSION TITLE
Support of using Helm

### DISCUSSION THREAD

**Discussion by sp185503** [argoproj/argo-rollouts#898] [Q&A] [answered]
Hi,

From what I've read most examples with Argo Rollouts illustrate the use of Kustomize for templating.  Does this indicate that the use of Helm with Argo Rollouts is not possible?

**Comment by carlosdoordash** [comment]
It is totally possible to use Argo Rollouts with Helm. We have a complex helm chart that allows service teams to choose either a RollingUpdate Kuberenetes Deployment resource or a Blue/Green Argo Rollout resource. The Helm chart hides the complexity behind simple values and both the Deployment and Rollout resources share the same underlying Pod Template.

Basically, the templating engine is irrelevant as long as the Kubernetes resource being applied is an Argo Rollout.

**Comment by kostis-codefresh** [comment, marked as answer]
Documentation was updated to make it clear that Argo Rollouts supports Helm deployments https://argoproj.github.io/argo-rollouts/features/helm/

**Reply by karthilinux91** [reply]
Thanks!
